### PR TITLE
Use Qt/Jolla coding style

### PR DESCRIPTION
**Pull request by pvuorela**
Greetings from the bike shed.

### CODE REVIEW

**Comment by pvuorela** [comment]
@rainemak I think this was your request.

**Comment by SfietKonstantin** [comment]
Did you run an automated tool or did you edited it by hand ?
Might be good to publish the Jolla QML style, and Jolla C++ style

**Comment by pvuorela** [comment]
Did this by hand. Not aware of such tool for qml and too small amount of c++ to bother adjusting automatic tool setup. 

C++ is mostly https://wiki.qt.io/Qt_Coding_Style , qml then adapting that plus explicitly avoiding semicolons. Would be good to document, but I think that'll happen when there are more components open sourced.

File: models/documentlistmodel.cpp [hunk 1]
```diff
@@ -60,27 +60,26 @@ DocumentListModel::~DocumentListModel()
...
-            return d->entries.at( index.row() ).documentClass;
-        default:
-            break;
+    switch (role) {
+    case FileNameRole:
+        return d->entries.at( index.row() ).fileName;
+    case FilePathRole:
+        return d->entries.at( index.row() ).filePath;
+    case FileTypeRole:
+        return QString().append(d->entries.at( index.row() ).fileType).append(d->entries.at( index.row() ).fileRead.toString(Qt::ISODate));
```
Comment (rainemak) [inline models/documentlistmodel.cpp]: Extra spaces in ```at( index.row() )```. Also string builder or arg could be used for building file the file type role.

File: models/documentlistmodel.h [hunk 2]
```diff
@@ -54,8 +54,8 @@ class DocumentListModel : public QAbstractListModel
     DocumentListModel( const DocumentListModel& ) = delete;
```
Comment (rainemak) [inline models/documentlistmodel.h]: Extra spaces.

File: models/documentlistmodel.h [hunk 3]
```diff
@@ -54,8 +54,8 @@ class DocumentListModel : public QAbstractListModel
...
     DocumentListModel& operator=( const DocumentListModel& ) = delete;
 
-    virtual QVariant data(const QModelIndex& index, int role) const;
-    virtual int rowCount(const QModelIndex& parent) const;
+    virtual QVariant data(const QModelIndex &index, int role) const;
+    virtual int rowCount(const QModelIndex &parent) const;
     virtual QHash< int, QByteArray > roleNames() const;
```
Comment (rainemak) [inline models/documentlistmodel.h]: Extra spaces... See also line 70.

File: dbusadaptor.cpp [hunk 4]
```diff
@@ -30,14 +30,11 @@ DBusAdaptor::~DBusAdaptor()
 {
```
Comment (rainemak) [inline dbusadaptor.cpp]: This comment belongs to line 25:

I'd prefer
```
DBusAdaptor::DBusAdaptor(QQuickView* view)
    : QDBusAbstractAdaptor(view)
    , m_view(view)
 {
 }
```
See other constructor initialization lists for elsewhere as well. Seems that some constructors use above style some not.

File: models/documentproviderlistmodel.cpp [hunk 5]
```diff
@@ -154,59 +154,56 @@ QModelIndex DocumentProviderListModel::index(int row, int column, const QModelIn
     return QAbstractListModel::index(row, column, parent);
 }
 
-QVariant DocumentProviderListModel::data(const QModelIndex& index, int role) const
```
Comment (rainemak) [inline models/documentproviderlistmodel.cpp]: See index method above this one as well.

File: models/documentproviderlistmodel.cpp [hunk 6]
```diff
@@ -154,59 +154,56 @@ QModelIndex DocumentProviderListModel::index(int row, int column, const QModelIn
...
 }
 
-int DocumentProviderListModel::rowCount(const QModelIndex& parent) const
+int DocumentProviderListModel::rowCount(const QModelIndex &parent) const
 {
-    if(parent.isValid())
+    if (parent.isValid())
         return 0;
     return d->providers.count();
 }
```
Comment (rainemak) [inline models/documentproviderlistmodel.cpp]: Extra spaces in roleNames() at line 211 (cannot add comment there).

File: models/documentproviderlistmodel.h [hunk 7]
```diff
@@ -50,7 +50,7 @@ class DocumentProviderListModel : public QAbstractListModel, public QQmlParserSt
         NeedsSetup
     };
 
-    explicit DocumentProviderListModel(QObject* parent = 0);
+    explicit DocumentProviderListModel(QObject *parent = 0);
```
Comment (rainemak) [inline models/documentproviderlistmodel.h]: See lines 65-67.

Comment (rainemak) [inline models/documentproviderlistmodel.h]: Funky private class name @ line 78 :)

File: models/trackerdocumentprovider.h [hunk 8]
```diff
@@ -59,11 +59,11 @@ public Q_SLOTS:
 
 private Q_SLOTS:
     void searchFinished();
-    void trackerGraphChanged(const QString& className, const QVariantList&, const QVariantList& );
```
Comment (rainemak) [inline models/trackerdocumentprovider.h]: See deleteFile at line 54.

File: pdf/pdfcanvas.cpp [hunk 9]
```diff
@@ -70,10 +70,10 @@ class PDFCanvas::Private
 
     int renderWidth;
 
-    PDFDocument* document;
+    PDFDocument *document;
     QQuickItem *flickable;
 
-    QTimer* resizeTimer;
+    QTimer *resizeTimer;
```
Comment (rainemak) [inline pdf/pdfcanvas.cpp]: Spaces in line 82

File: pdf/pdfcanvas.cpp [hunk 10]
```diff
@@ -167,7 +164,7 @@ void PDFCanvas::setDocument(PDFDocument* doc)
         connect( d->document, &PDFDocument::pageSizesFinished, this, &PDFCanvas::pageSizesFinished );
         connect( d->document, &PDFDocument::documentLocked, this, &PDFCanvas::documentLoaded );
 
-        if( d->document->isLoaded() )
+        if (d->document->isLoaded())
```
Comment (rainemak) [inline pdf/pdfcanvas.cpp]: Extra spaces at line 162-165.

File: pdf/pdfcanvas.cpp [hunk 11]
```diff
@@ -207,10 +204,9 @@ QColor PDFCanvas::linkColor() const
 
 void PDFCanvas::setLinkColor(const QColor& color)
 {
-    if( color != d->linkColor )
-    {
+    if (color != d->linkColor) {
```
Comment (rainemak) [inline pdf/pdfcanvas.cpp]: ```const QColor &color``` at line 205, see also 219.

File: pdf/pdfcanvas.cpp [hunk 12]
```diff
@@ -273,15 +266,12 @@ void PDFCanvas::layout()
...
-    {
-        const PDFPage& page = d->pages.value( i );
-        if( page.rect.contains( point ) )
-        {
-            for( const QPair< QRectF, QUrl >& link : page.links )
-            {
+    for (int i = 0; i < d->pageCount; ++i) {
+        const PDFPage& page = d->pages.value(i);
+        if (page.rect.contains(point)) {
+            for (const QPair< QRectF, QUrl >& link : page.links) {
```
Comment (rainemak) [inline pdf/pdfcanvas.cpp]: Extra spaces in QPair

File: pdf/pdfcanvas.cpp [hunk 13]
```diff
@@ -532,7 +504,7 @@ void PDFCanvas::resizeTimeout()
     update();
 }
 
-void PDFCanvas::pageSizesFinished(const QList< QSizeF >& sizes)
+void PDFCanvas::pageSizesFinished(const QList< QSizeF > &sizes)
```
Comment (rainemak) [inline pdf/pdfcanvas.cpp]: ```const QList<QSizeF> &sizes```

File: pdf/pdfdocument.h [hunk 14]
```diff
@@ -70,15 +70,15 @@ class PDFDocument : public QObject, public QQmlParserStatus
     virtual void componentComplete();
 
 public Q_SLOTS:
-    void setSource(const QString& source);
-    void requestUnLock( const QString& password );
-    void requestPage( int index, int size, QQuickWindow *window );
-    void prioritizeRequest( int index, int size);
+    void setSource(const QString &source);
```
Comment (rainemak) [inline pdf/pdfdocument.h]: See lines 41, 46, 51, 55, 59, 61

File: pdf/pdfjob.cpp [hunk 15]
```diff
@@ -58,9 +57,9 @@ void RenderPageJob::run()
 {
     Q_ASSERT(m_document);
 
-    Poppler::Page* page = m_document->page( m_index );
+    Poppler::Page *page = m_document->page( m_index );
```
Comment (rainemak) [inline pdf/pdfjob.cpp]: page(m_index);

File: pdf/pdfrenderthread.cpp [hunk 16]
```diff
@@ -79,8 +79,8 @@ class PDFRenderThreadPrivate
     Thread *thread;
 
     bool loadFailure;
-    Poppler::Document* document;
-    PDFTocModel* tocModel;
+    Poppler::Document *document;
```
Comment (rainemak) [inline pdf/pdfrenderthread.cpp]: Empty lines 68-69

File: plugin/DetailsPage.qml [hunk 17]
```diff
@@ -24,12 +24,12 @@ import org.kde.calligra 1.0 as Calligra
 Page {
     id: page;
 
-    property url source;
-    property int indexCount;
+    property url source
```
Comment (rainemak) [inline plugin/DetailsPage.qml]: See line 25

File: plugin/DocumentPage.qml [hunk 18]
```diff
@@ -23,25 +23,25 @@ import org.kde.calligra 1.0 as Calligra
 Page {
     id: base;
 
-    property variant attachedPage;
+    property variant attachedPage
 
-    property alias title: shareList.title;
-    property alias path: shareList.path;
-    property alias mimeType: shareList.mimeType;
+    property alias title: shareList.title
```
Comment (rainemak) [inline plugin/DocumentPage.qml]: See line 24

File: plugin/FileInfo.h [hunk 19]
```diff
@@ -35,7 +35,7 @@ class FileInfo : public QObject
     Q_PROPERTY(QDateTime modifiedDate READ modifiedDate NOTIFY sourceChanged)
 
 public:
-    explicit FileInfo(QObject* parent = 0);
+    explicit FileInfo(QObject *parent = 0);
```
Comment (rainemak) [inline plugin/FileInfo.h]: Could be part of Silica.

File: plugin/SpreadsheetPage.qml [hunk 20]
```diff
@@ -21,80 +21,80 @@ import Sailfish.Silica 1.0
...
     }
 
     Calligra.Document {
-        id: doc;
+        id: doc
     }
 
     Calligra.View {
-        id: v;
+        id: v
```
Comment (rainemak) [inline plugin/SpreadsheetPage.qml]: Not related to this PR but this variable name is bad the same applies to SilicaFlickable f.

File: plugin/ToolBar.qml [hunk 21]
```diff
@@ -24,7 +24,7 @@ PanelBackground {
 
     property variant flickable: undefined
```
Comment (rainemak) [inline plugin/ToolBar.qml]: Not related but this should be var instead of variant

File: plugin/ZoomableThumbnail.qml [hunk 22]
```diff
@@ -21,81 +21,78 @@ import Sailfish.Silica 1.0
...
-                    anchors.fill: parent;
-                    onClicked: base.clicked();
+                    anchors.fill: parent
+                    onClicked: base.clicked()
                 }
             }
         ]
     }
 
     QtObject {
```
Comment (rainemak) [inline plugin/ZoomableThumbnail.qml]: I'd get rid of this "private" object.

File: qml/CoverPage.qml [hunk 23]
```diff
@@ -87,30 +87,30 @@ CoverBackground {
         Image {
             id: previewImage
             anchors.fill: parent
-            property QtObject coverWindow;
-            property bool isGrabAvailable: typeof previewImage.grabToImage !== 'undefined';
+            property QtObject coverWindow
```
Comment (rainemak) [inline qml/CoverPage.qml]: Spaces missing @ line 41

File: plugin/ZoomableThumbnail.qml [hunk 24]
```diff
@@ -21,81 +21,78 @@ import Sailfish.Silica 1.0
...
-        thumb.width *= amount;
-        updateTimer.restart();
+        thumb.width *= amount
+        updateTimer.restart()
 
-        if(thumb.width < d.minWidth) {
-            thumb.width = d.minWidth;
+        if (thumb.width < d.minWidth) {
+            thumb.width = d.minWidth
         }
```
Comment (rainemak) [inline plugin/ZoomableThumbnail.qml]: Curly brackets not necessary.

File: plugin/ZoomableThumbnail.qml [hunk 25]
```diff
@@ -21,81 +21,78 @@ import Sailfish.Silica 1.0
...
 
-        if(thumb.width > d.maxWidth) {
-            thumb.width = d.maxWidth;
+        if (thumb.width > d.maxWidth) {
+            thumb.width = d.maxWidth
         }
```
Comment (rainemak) [inline plugin/ZoomableThumbnail.qml]: Same as above.

**Comment by rainemak** [comment]
Added some minor comments. Probably missed some but generally speaking LGTM.

File: dbusadaptor.cpp (hunk 4 above)
Comment (pvuorela) [inline dbusadaptor.cpp]: Undefined with Qt conventions and IMO irrelevant.

File: plugin/FileInfo.h (hunk 19 above)
Comment (pvuorela) [inline plugin/FileInfo.h]: I think it's a bit out of scope as non UI element. Additionally haven't seen similar implementations, so might not be that needed.

File: plugin/SpreadsheetPage.qml (hunk 20 above)
Comment (pvuorela) [inline plugin/SpreadsheetPage.qml]: Agreed and was about to change while at it. Didn't instantly figure out a better so didn't do that yet. Could continue later with more cleanups.

**Pull request merged**